tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -42,3 +44,69 @@ Feature: spaces management
 #      | id     |
 #      | team.b |
 #    And "Brian" logs out
+  Scenario: multiple spaces can be managed at once in the admin settings
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+      | Brian |
+    And "Admin" assigns following roles to the users
+      | id    | role       |
+      | Alice | SpaceAdmin |
+      | Brian | SpaceAdmin |
+    When "Alice" logs in
+    And "Alice" opens the "files" app
+    And "Alice" navigates to the projects space page
+    And "Alice" creates the following project spaces
+      | name   | id     |
+      | team A | team.a |
+      | team B | team.b |
+      | team C | team.c |
+      | team D | team.d |
+      | team E | team.e |
+      | team F | team.f |
+      | team G | team.g |
+      | team H | team.h |
+      | team I | team.i |
+      | team J | team.j |
+    And "Alice" opens the "admin-settings" app
+    And "Alice" navigates to the project spaces management page
+    And "Alice" disables the following spaces using the batch-actions
+      | name   |
+      | team.a |
+      | team.b |
+    And "Alice" disables the following spaces using the batch-actions
+      | name   |
+      | team.a |
+      | team.b |
+      | team.c |
+      | team.d |
+      | team.e |
+      | team.f |
+      | team.g |
+      | team.h |
+    Then "Alice" should still have selected the spaces and the batch-action disable gone
```

Comment:
Not sure if this step is really necessary. I know that's an acceptance criteria, but tests like "UI shows/doesn't show element xy" is stuff for unit-tests IMO. But I might be wrong here 🤷

Reply:
@kulmann What do you think about this step? Should we leave it?

Reply:
everything that can be a unit test is IMO not needed in e2e tests.